docs/source/index.rst

```diff
@@ -33,6 +33,11 @@ Colab
 
 Play with the `Colab notebook <https://bit.ly/omnizart-colab>`_ to transcribe your favorite song almost immediately!
 
+Replicate web demo
+*****
+
+Transcribe music with a simple web UI on  `Replicate <https://replicate.com/breezewhite/omnizart>`_.
```

Comment:
I think this sentence can be rephrased to "Transcribe music with \`Replicate web UI <https://replicate.com/breezewhite/omnizart>\`_."